server/extension/extension-common-flyway/src/main/resources/db/migration/V4__230322_ADD_workspace_COLUMN.sql

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 Holoinsight Project Authors. Licensed under Apache-2.0.
+ */
+DROP PROCEDURE IF EXISTS Uniform_Holoinsight_Column;
+DELIMITER $$
+-- 1 ADD COLUMN,2 UPDATE COLUMN,3 DELETE COLUMN
+CREATE PROCEDURE Uniform_Holoinsight_Column(TableName VARCHAR(50),ColumnName VARCHAR(50),SqlStr VARCHAR(4000),CType INT)
```

Comment:
Why use a PROCEDURE here ?
It should be plain DDL SQL here.

Reply:
Using stored procedure, can determine whether fields exist or not to avoid an alter sql execution failure.